_about/employment-opportunities.html

```diff
@@ -0,0 +1,29 @@
+---
+title: Employment Opportunities
+---
+
+<p class="mt-3">
+    The OSG Consortium is proud to display the following positions dedicated to advancing the state of research computing.
+</p>
+
+{% include /get/jobs.liquid %}
+
+{% assign internal_jobs = jobs | where_exp: "x", "x.type == 'internal'" %}
+{% if internal_jobs.size > 0 %}
+    {% for job in internal_jobs %}
+        {% include /cards/job.html %}
+    {% endfor %}
+{% endif %}
+
+{% assign external_jobs = jobs | where_exp: "x", "x.type == 'external'" %}
+{% if external_jobs.size > 0 %}
+    <h2 class="mt-4" id="external-positions">OSG Affiliate Institution Positions</h2>
+    <p>
+        The following positions are offered by our affiliate institutions, and seek to address a similar goal of advancing the
```

Comment:
Something like, “The following positions may be of interest to those in the research computing field, but are not affiliated with the OSG program of work nor are endorsed or recommended by OSG leadership.”